### PR TITLE
Refactor/tests

### PR DESCRIPTION
**Pull request by yiranli624**
#### What's this PR do?
Refactor cypress tests mainly, remove cypress.request, hitting actual server
Add delete at the end of the eventView test, so it removes the post and patch that was created during test

#### What does it fix?
n/a
#### Where should the reviewer start?
EventView_test.js Line 7;
Postings_test.js  Line 5;
User_test.js  Line 4;
App.test.js Line 2;
PostingView/index.js  Line 14;
src/apiCalls.js Line 34;

#### How should this be manually tested?
run ```npm run cypress``` and ```npm test```

### CODE REVIEW

File: cypress/integration/EventView_test.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ describe("Testing the single event details page", () => {
     cy.visit("http://localhost:3000/postings/event-1");
   })
 
+  afterEach(() => {
```
Comment (okalimbet) [inline cypress/integration/EventView_test.js:9]: Nice catch, it helped with the tests!

File: src/apiCalls.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ const getData = (path) => {
 };
 
 const updateData = (path, action, data) => {
+  console.log(path,action);
```
Comment (okalimbet) [inline src/apiCalls.js]: Remove console.log!

File: src/apiCalls.js [hunk 3]
```diff
@@ -10,9 +10,7 @@ const getData = (path) => {
   });
 };
 
-const updateData = (path, action, data) => {
-  console.log(path,action);
-  
+const updateData = (path, action, data) => {
```
Comment (okalimbet) [inline src/apiCalls.js:13]: Thank you ahaha!

**Review by okalimbet — APPROVED** [review]

File: src/apiCalls.js (hunk 2 above)
Comment (yiranli624) [reply src/apiCalls.js]: oops! Done!

**Pull request merged**